guides/common/modules/proc_generating-metadata-and-listing-files-for-syncable-exports.adoc

```diff
@@ -0,0 +1,26 @@
+[id="generating-metadata-and-listing-files-for-syncable-exports_{context}"]
+= Generating metadata and listing files for syncable exports
+
+In most cases, when exporting content in a syncable format using root privileges and the standard commands, metadata and listing files are generated automatically. However, you must manually trigger the generation of these files in the following circumstances:
+
+* When a non-root user is doing the export.
+* When the `--async` flag has been passed, for root or non-root user.
+* When triggering API manually or using Ansible modules to do the exports.
+
+.Prerequisites
+* The syncable export directory is accessible on disk.
+
+.Procedure
+. To generate metadata for a syncable export that was performed asynchronously or if the metadata was lost, use the following command:
```

Comment:
```suggestion
. Generate metadata for a syncable export:
```

suggestion (non-blocking): reword

* I'd drop the "use the following command" because that is always the case.
* I'd shorten it because the "that" and "if" clause almost duplicates the information/list of reasons to do this in the beginning.